data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/GrokLogstashPluginAttributesMapper.java

```diff
@@ -66,20 +66,22 @@ protected void mapCustomAttributes(List<LogstashAttribute> logstashAttributes, L
             if (logstashGrokMatchValueType.equals(LogstashValueType.HASH)) {
```

Comment:
Not related to this PR, we should consider refactoring the if-else block. We can create an interface and implementation for specific type of match attribute (HASH vs ARRAY vs Others...) to follow single responsibility principle and keep code maintainable.